File: app/src/main/java/com/example/myanimedata/api/MangaResponseDetail.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class MangaResponseDetail {
    @SerializedName("data")
    private final MangaDetail mangaDetails;

    public MangaDetail getMangaDetails() {
        return mangaDetails;
    }

    public MangaResponseDetail(MangaDetail mangaDetails) {
        this.mangaDetails = mangaDetails;
    }
}

File: app/src/main/java/com/example/myanimedata/api/MangaResponseDetail.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class MangaResponseDetail(
    @field:SerializedName("data") val mangaDetails: MangaDetail? = null
) : Parcelable